### DISCUSSION TITLE
openid-client with Nuxt3

### DISCUSSION THREAD

**Discussion by LamaDabbeet** [panva/openid-client#595] [Q&A] [answered]
hello, is openid-client supporting Nuxt3 , have anyone tried this before using Fusion Auth as identity provider ?

**Comment by panva** [comment]
https://github.com/panva/node-openid-client#how-do-i-use-it-outside-of-nodejs

**Comment by panva** [comment, marked as answer]
https://github.com/panva/openid-client/releases/tag/v6.0.0
